docs/domains/Domain.json

```diff
@@ -183,6 +183,14 @@
           "description": "Model in image model files and properties.",
           "$ref": "#/definitions/Model"
         },
+        "distributionStrategy": {
+          "description": "The strategy for handling changes to changes to the secrets or overridesConfigMap fields. Legal values are DYNAMIC and ROLLING.",
```

Comment:
duplicate "changes to"

Reply:
Thanks - actually generated from a comment on Configuration.java